source/GlassView.Core/Serialization/BenchmarkSerializer.cs

```diff
@@ -0,0 +1,142 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Atmoos.GlassView.Core.Models;
+
+using JR = System.Text.Json.Utf8JsonReader;
+
+using static System.Text.Json.JsonTokenType;
+
+namespace Atmoos.GlassView.Core.Serialization;
+
+// ToDo #10: Remove when upstream issue is resolved.
+// --> https://github.com/dotnet/runtime/issues/63791
+internal sealed class BenchmarkSerializer : JsonConverter<BenchmarkSummary>
+{
+    private const String benchmarkCases = "BenchmarkCases";
+
+    public override void Write(Utf8JsonWriter writer, BenchmarkSummary value, JsonSerializerOptions options)
+    {
+        writer.WriteStartObject();
+        WriteBuiltInTypes(new JsonWriter(writer), value);
+        WriteGlassViewTypes(writer, value, options);
+        writer.WriteEndObject();
+
+        static void WriteBuiltInTypes(JsonWriter writer, BenchmarkSummary value)
+        {
+            writer.Write(nameof(value.Name), value.Name);
+            writer.Write(nameof(value.Count), value.Count);
+            writer.Write(nameof(value.Namespace), value.Namespace);
+            writer.Write(nameof(value.Timestamp), value.Timestamp);
+            writer.Write(nameof(value.Duration), value.Duration);
+        }
+
+        static void WriteGlassViewTypes(Utf8JsonWriter writer, BenchmarkSummary value, JsonSerializerOptions options)
+        {
+            writer.WritePropertyName(nameof(value.Environment));
+            JsonSerializer.Serialize(writer, value.Environment, options);
+            writer.WritePropertyName(benchmarkCases);
+            JsonSerializer.Serialize(writer, (IEnumerable<BenchmarkCase>)value, options);
+        }
+    }
+
+    public override BenchmarkSummary? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        Int32 readProperties = 0;
+        const Int32 properties = 7;
+        var builder = new SummaryBuilder();
+        var depth = reader.CurrentDepth;
+
+        while (readProperties++ < properties) {
```

Comment:
Please place the paranthesis on a new line. OR run `dotnet format` which will align all files to editorconfig.
If there are too many changes, `dotnet format whitespace` should suffice.

Reply:
This is actually specified in my [.editorconfig](https://github.com/atmoos/GlassView/blob/main/source/.editorconfig) file:

```txt
csharp_new_line_before_open_brace = methods, properties, types, lambdas
```
I personally much prefer keeping the pranthesis `{` on the same line as control flow statements.

Let's resolve this with issue #14.